EnglishWebRoot/js/viewSubject.js

```diff
@@ -239,23 +241,27 @@ document.addEventListener("deviceready", function () {
         }
         var ret = {};
         ret.cols = [];
-        ret.cols.push({gravity: 0.05});
+        ret.cols.push({ gravity: 0.05 });
         var attrs = "";
-        // ToDo: Add Hindi content tag in school.xml
-        var subjectName = ["पढ़ाई और लिखाई","गणित"];
+        var subjectName = ["पढ़ाई और लिखाई", "गणित"];
         // Considering the English content from the EnglishWebRoot, overiding the content to the Hindi language
         // Note: We have not updated content in school.xml file as the property is referenced elsewhere.
         for (var a = 0; a < additionalAttributes.length; a++) {
             attrs += " " + additionalAttributes[a][0] + "='" + additionalAttributes[a][1] + "'";
         }
-        if (name === "Reading and Writing"){
+        var englishName;
+        if (name === "Reading and Writing") {
+            englishName = name;
             name = subjectName[0];
         }
         else if (name === "Math") {
+            englishName = name;
             name = subjectName[1];
         }
-        ret.cols.push({height: window.barHeight, template: "<div class='blueBox " + locked + "' id=subjectHeader " + attrs + "><span></span><img src='img/Button_Play.png' /><h2>" + name + "</h2></div>"});
-        ret.cols.push({gravity: 0.05});
+        // Setting the subject name in Hindi and English
+        ret.cols.push({ height: window.barHeight, template: "<div class='blueBox " + locked + "' id=subjectHeader " + attrs + "><span></span><img src='img/Button_Play.png' style='height:120%'/><h2 style='margin-top:-6% ; font-size:200%'>" + englishName + "</h2><h2 style='display: block ; transform: translate(6%,0); margin-top:-6% ; font-size:200%'>" + name + "</h2></div>" });

```

Comment:
Can we add the styles in SubjectPage.css?

Reply:
I have checked it, but it has some problem